src/main/java/net/syscon/elite/api/resource/impl/UserResourceImpl.java

```diff
@@ -198,23 +211,14 @@ public UserDetail getUserDetails(final String username) {
 
     @Override
     public ResponseEntity<List<OffenderBooking>> getMyAssignments(final Long pageOffset, final Long pageLimit) {
-        final var nomisProfile = ProfileUtil.isNomisProfile(env);
         var iepLevel = false;
-        List<Long> bookingIds = null;
+        List<Long> bookingIds = inmateService.getPersonalOfficerBookings(authenticationFacade.getCurrentUsername());
         List<String> offenderNos = null;
 
-        if (nomisProfile) {
```

Comment:
Personal officers are not needed in Nomis - only in Syscon elite - we use keyworkers now in different service